example/index.js

```diff
@@ -435,6 +466,39 @@ function updateEnvBlur() {
 
 }
 
+
+function switchControls( src, dst ) {
+
+	dst.object.position.copy( src.object.position );
+	dst.target.copy( dst.target );
+	dst.zoom = src.zoom;
+	dst.enabled = true;
+	src.enabled = false;
+	dst.update();
+
+}
+
```

Comment:
Is it not possible to swap out the camera on the same controls object? I think ideally there would just be a single OrbitControls.

Reply:
First thing I tried, it didn't seem to work (OrbitControls is a bit of a weird beast in my experience). You're right though, it would be way better if there was just one, maybe it warrants some more hacking...